go.mod

```diff
@@ -18,6 +18,8 @@ require (
 	github.com/golang/protobuf v1.4.2
 	github.com/google/go-cmp v0.5.2
 	github.com/google/uuid v1.1.1
+	github.com/grpc-ecosystem/go-grpc-middleware/providers/zerolog/v2 v2.0.0-rc.2
+	github.com/grpc-ecosystem/go-grpc-middleware/v2 v2.0.0-20200501113911-9a95f0fdbfea
```

Comment:
using `version 2` if the `go-grpc-middleware` package since version 2 has the log interceptor for `zerolog` (version 1 doesn't)